pipenv/routines/install.py

```diff
@@ -463,7 +463,6 @@ def do_install_dependencies(
                 install_req, markers, req_line = install_req_from_pipfile(
                     req_name, pipfile_entry
                 )
-                req_line = f"{req_line}; {markers}" if markers else f"{req_line}"
```

Comment:
`install_req_from_pipfile` already takes into account the markers so this must have been another bug.